<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(9,36,43,255)</string>
  </property>
  <widget class="QLabel" name="name_Label2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SCANNING</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_Label1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);</string>
   </property>
   <property name="text">
    <string>MALWARE</string>
   </property>
  </widget>
  <widget class="QLabel" name="logo_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/icons8-stan-marsh-96.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgba(80,218,107,255);
color: rgb(255, 255, 255);
text-align: center;
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="enter_text_Label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ENTER TEXT OR FILE </string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_Button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string>BROWSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_Button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
color: rgb(255, 255, 255);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scan_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string>SCAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
color: rgb(255, 255, 255);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
border-color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="export_Button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>630</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string>EXPORT</string>
   </property>
  </widget>
  <widget class="QLabel" name="about_file">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>571</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="namefile_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>NAME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>SIZE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>DATE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>TYPE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_name_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="output_size_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="output_date_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="output_type_Label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>550</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>551</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_Label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>571</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scan_show_Label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SCAN Detail</string>
   </property>
  </widget>
  <zorder>name_Label1</zorder>
  <zorder>logo_Label</zorder>
  <zorder>name_Label2</zorder>
  <zorder>input_text</zorder>
  <zorder>enter_text_Label</zorder>
  <zorder>browse_Button</zorder>
  <zorder>clear_Button</zorder>
  <zorder>scan_Button</zorder>
  <zorder>back_Button</zorder>
  <zorder>export_Button</zorder>
  <zorder>about_file</zorder>
  <zorder>namefile_Label</zorder>
  <zorder>size_Label</zorder>
  <zorder>date_Label</zorder>
  <zorder>type_Label</zorder>
  <zorder>output_name_Label</zorder>
  <zorder>output_size_Label</zorder>
  <zorder>output_date_Label</zorder>
  <zorder>output_type_Label</zorder>
  <zorder>output_Label</zorder>
  <zorder>tableView</zorder>
  <zorder>scan_show_Label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
